platform/flight.go

```diff
@@ -16,13 +16,17 @@
 package platform
 
 import (
+	"fmt"
 	"sync"
+
+	"github.com/satori/go.uuid"
```

Comment:
Ignition has switched over to the [pborman/uuid](https://github.com/pborman/uuid) repo, should we switch here as well? See https://github.com/coreos/ignition/pull/560#issuecomment-398147683 for more info

Reply:
Probably, but `go.uuid` is used in a few places so that seems like a different PR.